src/blob_file_iterator.cc

```diff
@@ -29,8 +31,9 @@ bool BlobFileIterator::Init() {
     return false;
   }
   char footer_buf[BlobFileFooter::kEncodedLength];
-  status_ = file_->Read(file_size_ - BlobFileFooter::kEncodedLength,
-                        BlobFileFooter::kEncodedLength, &slice, footer_buf);
+  status_ =
+      file_->Read(file_size_ - BlobFileFooter::kEncodedLength,
```

Comment:
seems `for_gc_` is a hint for `RandomFileAccessReader` to decide whether to perform flow control?
Can we always pass `true` here cause `BlobFileIterator` is always used for GC?

Reply:
Didn't realize `BlobFileIterator` is only for GC. Updated.